doc/SAI-Proposal-PoE.md

```diff
@@ -0,0 +1,591 @@
+PoE support for SAI
+
+| Title       | PoE                    |
+| ----------- | ---------------------- |
+| Authors     | Daniela Murin, Marvell |
+| Status      | In review              |
+| Type        | Standards track        |
+| Created     | 02/25/2024             |
+| SAI-Version | 1.13                   |
+
+# Overview
+
+Power over Ethernet, or PoE, describes any of several standards that pass electric power along with data on twisted-pair Ethernet cabling. This allows a single cable to provide both a data connection and enough electricity to power networked devices such as wireless access points (WAPs), IP cameras and VoIP phones.
```

Comment:
Reframe the sentence as shown below:

Power over Ethernet, or PoE, describes any of the 802.3(AF, AT, BT) standards that deliver electrical power along with data on twisted-pair Ethernet cables. This allows a single cable to provide both data connection and electricity to power the networked devices such as wireless access points (WAPs), IP cameras and VoIP phones etc.

Reply:
Updated